spec/features/news_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'rails_helper'
+
+feature 'News' do
+  describe 'autirized as user' do
+    let(:user) { create :user }
+
+    before(:each) do
+      @news = FactoryGirl.create(:news, user_id: user.id)
+      sign_in user
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
+    end
+
+    scenario 'moves to news list page' do
+      visit news_index_path
+      expect(page).to have_content(@news.title)
+      expect(page).to have_content('Created at')
+      expect(page).not_to have_button('Add news')
+    end
+
+    scenario 'moves to chosen news page' do
+      visit news_index_path
+      click_link(@news.title)
+
+      expect(page).to have_content(@news.title)
+      expect(page).to have_content(@news.text)
+      expect(page).to have_content('Created at')
+      expect(page).not_to have_button('Edit')
+      expect(page).not_to have_content('Delete')
+    end
+  end
+
+  describe 'autorized as admin' do
+    let(:user) { create :user, role: 2 }
+
+    before(:each) do
+      @news = FactoryGirl.create(:news, user_id: user.id)
+      sign_in user
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
+    end
+
+    scenario 'moves to news list page' do
+      visit news_index_path
+      expect(page).to have_content(@news.title)
+      expect(page).to have_content('Created at')
+      expect(page).to have_button('Add news')
+    end
+
+    scenario 'moves to chosen news page' do
+      visit news_index_path
+      click_link(@news.title)
+
+      expect(page).to have_content(@news.title)
+      expect(page).to have_content(@news.text)
+      expect(page).to have_content('Created at')
+      expect(page).to have_button('Edit')
+      expect(page).to have_content('Delete')
+    end
+
+    scenario 'create new news' do
+      visit news_index_path
+      click_button 'Add news'
+      fill_in('Title', with: 'This is the test news!')
```

Comment:
@SBlazhko 
you have a factory
```ruby
FactoryGirl.define do
  factory :news do
    title Faker::ChuckNorris.fact
    text Faker::Hipster.paragraphs(5)
    news_image Faker::Avatar.image
  end
end

```
so you can use `attributes_for`. it returns hash of attributes. try it in a console.

Then

```ruby
let(:title) { attributes_for(:news)['title'] }
```

And this code returns news title